src/mbgl/tile/geometry_tile_worker.cpp

```diff
@@ -404,6 +410,17 @@ void GeometryTileWorker::parse() {
     requestNewImages(imageDependencies);
 
     performSymbolLayout();
+#if MBGL_TILE_TIMING
+    milliseconds parseEndAt = duration_cast< milliseconds >(system_clock::now().time_since_epoch());
+    Log::Debug(Event::HttpRequest,
```

Comment:
We should add a new event here, right? We might be able to just use something generic like `Event::Timing` for all timing logs?